### ISSUE TITLE
JSON encoding of at least a subset of ISO 19115

### ISSUE THREAD

**Issue by PeterParslow** [ISO-TC211/StandardsTracker#470]
Suggested at OGC API - Records code sprint in September 2022.

The OGC API - Records developers suggest standardising a JSON encoding of at least a subset of ISO 19115.

Some felt the encoding should cover the whole of ISO 19115, but I pointed out that ISO 19115 does (optionally) import a lot of other ISO/TC 211 standards.
One possible subset would be those aspects of ISO 19115 which map well to DCAT (see https://github.com/ISO-TC211/StandardsTracker/issues/449).
Another would be the metadata element set being defined by the OGC API - Records development (https://github.com/opengeospatial/ogcapi-records).

Discussion & experimentation at the code sprint suggested that "plain" JSON would be sufficient, although some considered that the presence of some geometry (even if just a bounding box) pointed towards GeoJSON. None felt that the extra capabilities of JSON-FG or JSON LD were necessary.

**Comment by wilkesg** [comment]
Hi Peter, sharing for awareness, if not already aware. Canada has a JSON encoded version of core ISO 19115. Colleagues at Natural Resources Canada (in the GeoDiscovery group which was previously The Federal Geospatial Platform) along with others have been implementing this in GeoCore available on Git. 
See  https://github.com/Canadian-Geospatial-Platform/geocore
https://canadian-geospatial-platform.github.io/geocore/
https://github.com/Canadian-Geospatial-Platform/STAC_to_GeoCore
If there is interest to learn more we can make put you in touch. 
Cheers, 
Graham

**Comment by PeterParslow** [comment]
Notes from discussion in JAG, 2022-12-06:

- OGC has some drafts / workable JSON encodings of a few ISO standards, including 19115-1, within Testbed Engineering reports
- DGIWG has a requirement for this, focussed on the STANAG profile of ISO 19115. They're also interested to embed metadata in GeoPackage
- Could be ISO 19115-4? Could be a TR?
- To be discussed in PMG, not least because ISO 19115 is overseen by WG7, and OGC API Records may (would logically "belong") to WG4.

**Comment by ejbleys** [comment]
Just wondering: as the ISO 19115-4 would have JSON schemas for ISO 19115-1/2 should they reside somewhere near the publicly available resources for XML schemas (https://github.com/ISO-TC211/schemas.isotc211.org)?

**Comment by ronaldtse** [comment]
I would think that XML schemas and JSON schemas should both be under the schemas site?

**Comment by wilkesg** [comment]
Agree, think it would make sense to keep them under the same site.
